core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -1869,8 +1869,7 @@ public void _run() throws IOException, InstallationStatus {
                 // Do this first so we can avoid duplicate downloads, too
                 // check to see if the plugin is already installed at the same version and skip it
                 LOGGER.info("Skipping duplicate install of: " + plugin.getDisplayName() + "@" + plugin.version);
-                //throw new Skipped(); // TODO set skipped once we have a status indicator for it
-                return;
+                throw new Skipped();
```

Comment:
Well I didn't check other builds -- we'd have heard about this by now if it was released -- but my guess is the same. But may be as simple as fixing some math in the wizard.

That said, I wonder whether it's useful to even show this. Skipped duplicate installs are an artifact of how dependency resolution works and I don't really see what value admins get from this (i.e. knowing that more than one selected for install plugin depend on some other plugin that already got installed -- AFAIU).